src/Persistence/AbstractQuery.php

```diff
@@ -328,16 +438,30 @@ protected function executeQueryWithDebug(\Closure $fx)
         }
     }
 
+    public function getIterator(): iterable
+    {
+        return $this->execute();
+    }
+
+    /**
+     * Return the model the query runs on.
```

Comment:
comments like this does not provide any value, they only distract or even confuse if formulated not-up-too date - remove